modules/evaluation/src/main/Assessible.scala

```diff
@@ -117,6 +117,13 @@ case class Assessible(analysed: Analysed, color: Color) {
   lazy val mtSd: Int  = listDeviation(~game.moveTimes(color) map (_.roundTenths)).toInt
   lazy val blurs: Int = game.playerBlurPercent(color)
 
+  lazy val tcFactor: Double = game.speed match {
+    case Speed.Bullet | Speed.Blitz => 1.25
+    case Speed.Rapid => 1.0
+    case Speed.Classical => 0.6
+    case _ => 1.0
```

Comment:
ultrabullet?

Reply:
ultrabullet probably fine as 1.0 -- https://github.com/ornicar/lila/blob/master/modules/evaluation/src/main/Assessible.scala#L51 makes that only accuracy and hold alert really matters and I haven't seen a need of an increased factor there

Reply:
should it be put explicitly in the list though, like Rapid, or nah?